test/src/test/java/hudson/security/SecurityRealmTest.java

```diff
@@ -79,4 +82,62 @@ public boolean validateCaptcha(String id, String text) {
         public void generateImage(String id, OutputStream ios) throws IOException {
         }
     }
+
+    static void addSessionCookie(CookieManager manager, String domain, String path, Date date) {
+        manager.addCookie(new Cookie(domain, "JSESSIONID."+Integer.toHexString(new Random().nextInt()),
+                "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+                path,
+                date,
+                false));
+    }
+
+    @Test
+    public void many_sessions_logout() throws Exception {
+        final String WILL_NOT_BE_SENT = "/will-not-be-sent";
+        final String LOCALHOST = "localhost";
+        final String JSESSIONID = "JSESSIONID";
+
+        JenkinsRule.WebClient wc = j.createWebClient();
+        CookieManager manager = wc.getCookieManager();
+        manager.setCookiesEnabled(true);
+        wc.goTo("login");
+
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.DAY_OF_YEAR, 1);
+        Date tomorrow = calendar.getTime();
+        byte[] array = new byte[7];
```

Comment:
unused variable
```suggestion
```

(⚠️ first time I tried the suggestion feature to remove a line)